__tests__/app.test.js

```diff
@@ -26,6 +26,25 @@ describe('GET /api/articles/:article_id', () => {
             });
         });
     });
+    test('REFACTORED: return an article object by id and + comment_count ', () => {
```

Comment:
Thank you for denoting this has been refactored, but please remove "REFACTORED" from your final repo 👍 .

Reply:
ok got it :)